Sources/Clickstream/Network/NetRequest.swift

```diff
@@ -32,7 +32,8 @@ enum NetRequest {
             URLQueryItem(name: "platform", value: "iOS"),
             URLQueryItem(name: "appId", value: configuration.appId),
             URLQueryItem(name: "compression", value: compression),
-            URLQueryItem(name: "event_bundle_sequence_id", value: String(describing: bundleSequenceId))
+            URLQueryItem(name: "event_bundle_sequence_id", value: String(describing: bundleSequenceId)),
+            URLQueryItem(name: "hashCode", value: requestData.hashCode())
```

Comment:
add a test case to verify the value against the event body

Reply:
added test case.